src/redux/news-detail/news-detail.reducer.js

```diff
@@ -0,0 +1,36 @@
+import { SET_ARTICLE } from './news-detail.types';
+
+const initialState = {
+  item: {
+    title: [{ value: 'завантаження' }, { value: 'loading' }],
```

Comment:
Why do we need this code? I think actual news item should be empty until it is fetched from backend

Reply:
made it an empty string